tests/sensors/test_base_sensor.py

```diff
@@ -464,3 +464,43 @@ def test_reschedule_with_test_mode(self):
                 self.assertEqual(len(task_reschedules), 0)
             if ti.task_id == DUMMY_OP:
                 self.assertEqual(ti.state, State.NONE)
+
+    def test_sensor_with_invalid_poke_interval(self):
+        negative_poke_interval = -10
+        non_number_poke_interval = "abcd"
+        positive_poke_interval = 10
+        with self.assertRaises(AirflowException):
+            self._make_sensor(
+                return_value=None,
+                poke_interval=negative_poke_interval,
+                timeout=25)
+
+            self._make_sensor(
+                return_value=None,
+                poke_interval=non_number_poke_interval,
+                timeout=25)
```

Comment:
Sorry for having missed to point this out: I think you should have another `with self.assertRaises(AirflowException):` for the 2nd check on `non_number_poke_interval`.

When you run with `negative_poke_interval`, the `assertRaises` is already triggered and this chunk will stop. So your check on `non_number_poke_interval` is not really running.